### DISCUSSION TITLE
MPC Autotune: How to change temp target

### DISCUSSION THREAD

**Discussion by Micah-Scheck** [mriscoc/Ender3V2S1#1059] [Q&A]
Hello, I just printed a new fan mount for my e3v2 neo and I can't get the silicone sock to fit making the hotend lose temp when the fan turns on. I mostly print in abs at 245-250C so the tune targeting 200c isn't very helpful for me. Is there a way to change the temperature target of the test so I can hopefully fix this?

**Comment by mriscoc** [comment]
MPC is a model, so the routine only needs to calculate the parameters for a given temperature and extrapolate the model to the full range of working temperatures.
